### DISCUSSION TITLE
Connessione da remoto

### DISCUSSION THREAD

**Discussion by hpienergy** [indomus/forum#3033] [HUB: Home Assistant]
Buongiorno,
chiedo cortesemente una informazione sulla connessione da remoto a Homeassistant: è possibile tramite l'app di Homeassistant android puntare direttamente l'indirizzo ip del router, l'ip pubblico statico, con la porta aperta che manda all'ip locale di Homeassistant? Senza usare servizi tipo duckdns cloudflare ecc... O ci sono dei blocchi che non lo consentono? Saluti

**Comment by snakuzzo** [comment]
Devi per forza usare un FQDN.
Questo perché, in particolare per l'app, per la connessione in HTTPS è necessario un certificato rilasciato da una CA valida. E le CA solitamente rilasciano i certificati solo in base al FQDN e non all'indirizzo IP.

**Reply by hpienergy** [reply]
Capito, però l'app di Homeassistant può usare l'http come tipo connessione? O ci sono dei blocchi che non lo consentono?

**Reply by snakuzzo** [reply]
Mi sembra di sì, ma esporre un servizio del genere in HTTP è una scelta completamente sbagliata (e che sia chiaro...non è che lo dico io)

**Reply by hpienergy** [reply]
Certo in HTTP è rischioso, però in emergenza se i servizi di autentificazione non funzionano è l'unica via più immediata, secondo me dovrebbe essere a discrezione dell'utilizzatore il livello di sicurezza da usare.

**Reply by snakuzzo** [reply]
Forse intendi autenticazione...ma per il resto ti giuro che non ho capito che intendi dire e cosa c'entra tutto questo con la domanda iniziale.
Inoltre, HA prevede autenticazione e questa fa parte del core...non c'è alcun servizio che può o non può funzionare.
Scusami se te lo dco, ma stai andando proprio fuori strada con questo ragionamento

**Comment by trebestie** [comment]
Non ho mai provato, ma non vedo perchè non dovrebbe essere possibile

**Comment by hpienergy** [comment]
![Screenshot_20240410_151719](https://github.com/indomus/forum/assets/62174979/3d59ad6e-d292-4fac-bd8c-a92d6cd206d8)

In http da remoto diretto in ip pubblico sulla porta aperta da errore e non si connette

**Comment by trebestie** [comment]
Mi correggo.
Nella documentazione c'è scritto che ci vuole una connessione ssl

https://companion.home-assistant.io/docs/troubleshooting/faqs

Il gio 11 apr 2024, 23:47 hpienergy ***@***.***> ha scritto:

**Reply by hpienergy** [reply]
Grazie per la risposta.
Alessandro
